tests/gtest-1.7.0/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." OFF)
 option(
   gtest_force_shared_crt
   "Use shared (DLL) run-time lib even when Google Test is built as static lib."
-  OFF)
+  ON)
```

Comment:
This flag was not being set even passing it to the cmake argument on parent cmake... Forcing it there.
Anyway we were forcing it in 2008 and 2010 also.